streamparse/cli/update_virtualenv.py

```diff
@@ -112,10 +112,12 @@ def create_or_update_virtualenvs(env_name, topology_name, options,
     storm_options = resolve_options(options, env_config, topology_class, topology_name)
     activate_env(env_name, storm_options, config_file=config_file)
 
+    virtualenv_flags = options.get('virtualenv_flags', env_config.get('virtualenv_flags'))
+
     # Actually create or update virtualenv on worker nodes
     execute(_create_or_update_virtualenv, env.virtualenv_root, virtualenv_name,
             requirements_paths,
-            virtualenv_flags=options.get('virtualenv_flags'),
+            virtualenv_flags=virtualenv_flags,
```

Comment:
Thanks for the speedy response @dan-blanchard!

I've tried out this suggestion, but it doesn't quite work as intended.
From what I can gather, when the prioritisation of configs is worked out [in `common.py`](https://github.com/Parsely/streamparse/blob/422d889b12d6dbbca347e74cd5e7fc88c225b658/streamparse/cli/common.py#L219), it's only the `options` part that gets applied to `storm_options`, therefore not capturing the virtualenv config such as `virtualenv_flags`.
(As a side note, I kinda took the name `storm_options` to mean config only relevant to Storm.)

I figured that since `virtualenv_flags` is the only argument that gets passed explicitly to `_create_or_update_virtualenv`, it would be easier to determine its value at that point instead of further complicating config resolution in `resolve_options`.
Would you suggest making `resolve_options` aware of more keys in the env config?